test/Microsoft.Health.Dicom.Tests.Integration/Persistence/DeleteServiceTests.cs

```diff
@@ -88,8 +88,8 @@ private async Task ValidateRemoval(bool success, int retrievedInstanceCount, Ver
         Assert.True(success);
         Assert.Equal(1, retrievedInstanceCount);
 
-        await Assert.ThrowsAsync<ItemNotFoundException>(async () => await _fixture.MetadataStore.GetInstanceMetadataAsync(versionedInstanceIdentifier));
-        await Assert.ThrowsAsync<ItemNotFoundException>(async () => await _fixture.FileStore.GetFileAsync(versionedInstanceIdentifier));
+        await Assert.ThrowsAsync<ItemNotFoundException>(async () => await _fixture.MetadataStore.GetInstanceMetadataAsync(versionedInstanceIdentifier.Version));
```

Comment:
nit: while you're here you can remove `async` and `await` from this lambda

Reply:
Done